lms/djangoapps/teams/csv.py

```diff
@@ -136,25 +136,19 @@ def add_user_to_team(self, user_row, course):
                     # teamsets
                     team = CourseTeam.objects.get(name=team_name, topic_id=self.teamset_index_dictionary[i])
                 except CourseTeam.DoesNotExist:
-                    # course_module = modulestore().get_course(course.id)
                     team = CourseTeam.create(name=team_name, course_id=course.id, description='Import from csv',
                                              topic_id=self.teamset_index_dictionary[i]
                                              )
                     team.save()
-
-                # if not has_team_api_access(request.user, team.course_id, access_username=username):
-                #     return Response(status=status.HTTP_404_NOT_FOUND)
-
                 # if not has_specific_team_access(request.user, team):
                 #    return Response(status=status.HTTP_403_FORBIDDEN)
```

Comment:
A nit pick of the first order:  Why leave these commented-out lines in?

Reply:
They should be removed in the latest version of the file.